scripts/cat/history.py

```diff
@@ -677,12 +676,13 @@ def reveal_murder(cat, other_cat, cat_class, victim, murder_index):
                 victim_history["revealed_by"] = other_cat.ID if other_cat else None
                 victim_history["revelation_moon"] = game.clan.age
                 if not other_cat:
-                    murder_history["revelation_text"] = \
-                        "The truth of {PRONOUN/m_c/poss} murder is known to the Clan."
+                    victim_history["revelation_text"] = \
+                        "The truth of their murder is known to the Clan."
                 else:
                     victim_history["revelation_text"] = \
-                        "The truth of {PRONOUN/m_c/poss} murder was discovered by [discoverer]."
+                        "The truth of their murder was discovered by [discoverer]."
```

Comment:
I'm not sure I understand why we're removing the pronoun tag?  Can you explain the reasoning? /gen

Reply:
Check my screenshots on #2393.  This text appears appended to the initial murder text.  I did change it to [victim] in my first commit, but realised the name didn't really need to be there.

Reply:
Hmm, this is still a pronoun that needs to be capable of being dynamic, though, isn't it?  It refers to the victim, so it needs to be able to change to the victim's pronouns, thus the pronoun tag is necessary.  Does "m_c" here refer to the murderer?  If so, we should change it to "r_c" or whatever it needs to be to refer to the victim cat.

Reply:
Seems it was correct the first time.  I think I get it now.  m_c is the main cat in the murder history, but the victim in the victim history.  When I was testing it the first time, I didn't realise both cats used them/them pronouns.  I changed the murderer to He and the victim to She and tried again.  I then found their history entries in the save files to see how they had been saved. 
![badgerpelt](https://github.com/ClanGenOfficial/clangen/assets/129736008/fd02fb6d-21d2-41c4-8539-774fe4ff102b)
![Shellblur](https://github.com/ClanGenOfficial/clangen/assets/129736008/1cd9b6b4-41c2-4598-87ea-e4ef4ee987dc)
![victimhistory](https://github.com/ClanGenOfficial/clangen/assets/129736008/de411258-d145-43b7-aaf2-974f0c217e95)